.gitignore

```diff
@@ -30,3 +30,4 @@ docs/_build
 package-lock.json
 webpack-stats.json
 lib/vendor
+app/test_*
```

Comment:
These aren't "tests" per se, but something that helps you debug or develop locally? If it could be useful to other folks, maybe it should be checked in, but your call.

Reply:
You're right, it is for local debugging. I usually have test_2.html and test_3.html and don't check them in because they change so often. They would be at best marginally useful and I'm afraid I'll end up polluting the commit log with ever-changing debug viewconfs.